src/main/java/org/jabref/logic/exporter/ExportFormat.java

```diff
@@ -313,12 +313,6 @@ public void performExport(final BibDatabaseContext databaseContext, final String
 
     }
 
-    @Override
-    public void performExport(final BibDatabaseContext databaseContext, Path file, final Charset encoding,
```

Comment:
Please don't delete this `path` based methods. I know they are not yet used but they are actually preferable to the string or file-based methods (these should actually be deprecated).

Reply:
I deleted it on purpose because it is not working at all as I explained in my other PR or earlier.
It always lead to an exception.

Reply:
ok...